Doxyfile

```diff
@@ -32,7 +32,7 @@ DOXYFILE_ENCODING      = UTF-8
 # title of most generated pages and in a few other places.
 # The default value is: My Project.
 
-PROJECT_NAME           = "Unity Buy SDK"
+PROJECT_NAME           = "Shopify SDK for Unity"
```

Comment:
Where is this used? Are we sure everything is safe with this change?

Reply:
Doxygen uses this as the title mostly from what I've seen.